### PR TITLE
fix #74888 animax.to

### LINKED ISSUE #74888: animax.to
### Issue URL (Ads)
 [https://animax.to/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fanimax.to%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 nightly 17
Browser: | Opera
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Block Push API, <br>Block Location API
DNS filtering: | server: `https://dns-unfiltered.adguard.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List
Custom filter subscriptions: | https://raw.githubusercontent.com/AdguardTeam/FiltersRegistry/master/filters/filter_15_DnsFilter/filter.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#74888

### CODE REVIEW

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
~~Hmm.. I thought I commented but apparently not~~ [EDIT] Found my comment. Add
`||animax.to/ax.js`
as `window._run` is defined by this. You can still keep the scriptlet, the site may change file path.

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -254,6 +254,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+animax.*##[src*="//a.exdynsrv.com/iframe.php"]
```
[the remark this answers is not in the document]
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:257]: I can reproduce it with only AdGuard App Do not why though.

File: EnglishFilter/sections/general_extensions.txt [hunk 2]
```diff
@@ -229,6 +229,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74888
+animax.*#%#//scriptlet("abort-on-property-read", "_run")
```
[the remark this answers is not in the document]
Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:233]: Ok sure.

File: EnglishFilter/sections/specific.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (Yuki2718) [reply EnglishFilter/sections/specific.txt:257]: What exactly is the AdGuard App? AdGuard for Windows?

File: EnglishFilter/sections/general_extensions.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (wout-codes) [reply EnglishFilter/sections/general_extensions.txt:233]: I will still keep the scriptlet.

File: EnglishFilter/sections/specific.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:257]: Yes

[the remark this answers is not in the document]
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:257]: I need more evidence then I will file an issue in AdGuard for Windows Bug Tracker Repository. My goal is to make the AdGuard App support `$popup` modifier and this placeholder issue.

**Review by Yuki2718 — APPROVED** [review]

**Comment by Yuki2718** [comment]
Whoops, `||animax.to/ax.js` should be in specific.txt

**Review by zloyden — APPROVED** [review]

**Pull request merged**